app/components/move_message.py

```diff
@@ -124,28 +326,335 @@ async def on_submit(self, interaction: discord.Interaction) -> None:
         )
 
 
-class DeleteOriginalMessage(discord.ui.View):
-    def __init__(self, message: discord.Message) -> None:
+class ChooseMessageAction(discord.ui.View):
+    attachment_button: discord.ui.Button[Self]
+    thread_button: discord.ui.Button[Self]
+    help_button: discord.ui.Button[Self]
+
+    def __init__(self, message: MovedMessage) -> None:
         super().__init__()
-        self.message = message
+        self._message = message
+        self._split_subtext = SplitSubtext(message)
+        self._attachment_button_added = False
+        # Adding the thread button might also add the attachment button so that
+        # the attachment button can be placed in between the thread and help
+        # buttons.
+        self._add_thread_button()
+        if not self._attachment_button_added:
+            self._add_attachment_button()
```

Comment:
`_add_thread_button` doesn't modify `_attachment_button_added`, which currently makes the variable useless if I'm reading this right.

Reply:
Right, I missed that!